.reuse/dep5

```diff
@@ -100,6 +100,10 @@ Files: internal/compiler/widgets/cupertino-base/_*.svg
 Copyright: Material Icons <https://github.com/material-icons/material-icons/blob/master/LICENSE>
 License: Apache-2.0
 
+Files: internal/compiler/widgets/qt/_*.svg
+Copyright: Material Icons <https://github.com/material-icons/material-icons/blob/master/LICENSE>
+License: Apache-2.0
+
```

Comment:
Ok, so I checked: this dep5 file has already an entry for widgets/qt/_*.svg that indicates that cosmic is the origin of the two svgs in there. That is correct - I copied from the cosmic style.


```suggestion
```